src/ECER.Resources.Documents/Applications/ApplicationRepository.cs

```diff
@@ -34,6 +34,7 @@ public async Task<IEnumerable<Application>> Query(ApplicationQuery query, Cancel
     context.LoadProperties(applications, ecer_Application.Fields.ecer_transcript_Applicationid);
     context.LoadProperties(applications, ecer_Application.Fields.ecer_workexperienceref_Applicationid_ecer);
     context.LoadProperties(applications, ecer_Application.Fields.ecer_characterreference_Applicationid);
+    context.LoadProperties(applications, ecer_Application.Fields.ecer_ecer_professionaldevelopment_Applicationi);
```

Comment:
hmm not sure if it's just a typo but _Applicationi ? <-- seems like it should be id? 

ignore if this is auto generated.